SwiftWisdom/Core/StandardLibrary/Numbers/Bool+Extensions.swift

```diff
@@ -1,33 +0,0 @@
-//
-//  Bool+Extensions.swift
-//  SwiftWisdom
-//
-//  Created by Logan Wright on 1/19/16.
-//  Copyright © 2016 Intrepid. All rights reserved.
-//
-
-import Foundation
-
-extension Bool {
-    /**
-     This function seeks to replicate the expected behavior of `var boolValue: Bool` on `NSString`.  Any variant of `yes`, `y`, `true`, `t`, or any numerical value greater than 0 will be considered `true`
-     */
-    public init(ip_string: String) {
```

Comment:
Since its trying to replicate `boolValue` from NSString, I changed it into a computed property in a String extension and handed it off to Alan for inclusion in his String update PR. We can do both if you think we need em.